Core/Controller.php

```diff
@@ -7,6 +7,14 @@ class Controller {
     protected function view($name, $params = null) {
         $varName = $name;
         $$varName = $params;
-        return require_once('App/views/' . $name . '.html');
+        /* This variable takes the entire file name and turns into a 
+        reverse array every space of the array is separated by a period "." */
+        $extension = array_reverse(explode('.', $name));
+
+        if ($extension[0] == 'html' or $extension[0] == 'php' or $extension[0] == 'phtml'){
+            return require_once('App/views/' . $name);
+        } else {
+            echo 'NOTICE: supported files are HTML, PHP and PHTML';
```

Comment:
Neste caso, melhor lançar uma exceção (erro) com throw, para que possa ser tratada. Pesquise sobre.
```suggestion
            throw new Exception('supported files are HTML, PHP and PHTML');
```
Porém, como toda essa lógica não está adequada, vale reesrever tudo.